src/elements/content-sidebar/MetadataInstanceEditor.tsx

```diff
@@ -31,18 +31,18 @@ const MetadataInstanceEditor: React.FC<MetadataInstanceEditorProps> = ({
     };
 
     return (
-        <AutofillContextProvider isAiSuggestionsFeatureEnabled={isBoxAiSuggestionsEnabled}>
-            <MetadataInstanceForm
-                isAiSuggestionsFeatureEnabled={isBoxAiSuggestionsEnabled}
-                isUnsavedChangesModalOpen={isUnsavedChangesModalOpen}
-                selectedTemplateInstance={template}
-                onCancel={handleCancel}
-                onSubmit={onSubmit}
-                setIsUnsavedChangesModalOpen={setIsUnsavedChangesModalOpen}
-                onDelete={onDelete}
-            />
-        </AutofillContextProvider>
+        <MetadataInstanceForm
+            areAiSuggestionsAvailable={isBoxAiSuggestionsEnabled}
+            isAiSuggestionsFeatureEnabled={isBoxAiSuggestionsEnabled}
```

Comment:
why to pass the same value in two different props?

Reply:
One says if the AI feature is enabled and `Autofill` button can be shown (`isAiSuggestionsFeatureEnabled`), and one if button can be active (`areAiSuggestionsAvailable`).

We want to have disabled `Autofill` button on files that are not supported by AI API.